tests/failpoints/cases/test_rawkv.rs

```diff
@@ -64,6 +64,25 @@ impl TestSuite {
         must_raw_put(&client, ctx, key.to_vec(), value.to_vec())
     }
 
+    pub fn raw_put_err(&mut self, key: &[u8], value: &[u8]) {
```

Comment:
Suggest:
1. name as `raw_put_err_by_timestamp_not_synced`, or
2. return error and verify outside, to make this method more common.

Reply:
I rename it with `raw_put_err_by_timestamp_not_synced`.